src/routes.js

```diff
@@ -39,11 +39,17 @@ import StandalonePostEditor from './containers/StandalonePostEditor'
 import Admin from './containers/Admin'
 import PageWithNav from './containers/PageWithNav'
 import TestBench from './containers/TestBench'
+import socketIOClient from 'socket.io-client'
+import sailsIOClient from 'sails.io.js'
 import { debug } from './util/logging'
 import { makeUrl } from './util/navigation'
 import { get, isEmpty, pick } from 'lodash'
 import config from './config'
 
+let io = sailsIOClient(socketIOClient)
```

Comment:
it feels cleaner to me to move this into App's `componentDidMount` and then pass it down through context